src/services/database.ts

```
@@ -0,0 +1,718 @@
+    if (!chat) return null;
+
+    const messages = await this.getMessages(chatId);
+    return { ...chat, messages };
+  }
+
+  static async searchChats(query: string): Promise<DatabaseChat[]> {
+    return this.chats.filter(chat => 
+      chat.title.toLowerCase().includes(query.toLowerCase())
+    );
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)